src/screens/FeedScreen/index.js

```diff
@@ -237,7 +236,7 @@ const FeedScreen = (props) => {
   };
 
   return (
-    <SafeAreaProvider style={styles.container} forceInset={{top: 'always'}}>
+    <View>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `SafeAreaProvider` to `View` might cause layout issues on devices with notches or rounded corners. The `SafeAreaProvider` component provides padding to avoid the notch and rounded corners, which is not provided by the `View` component.
```diff
- <View>
+ <SafeAreaProvider style={styles.container} forceInset={{top: 'always'}}>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->